TestProviders/AzureProvider.psm1

```diff
@@ -180,4 +180,18 @@ Class AzureProvider : TestProvider
 		}
 		return $false
 	}
+
+	[void] RunTestCleanup() {
+		# Wait till all the cleanup background jobs successfully started cleanup of resource groups.
+		$DeleteResourceGroupJobs = Get-Job | Where-Object { $_.Name -imatch "DeleteResourceGroup" }
```

Comment:
I remember we had a discussion that we don't want to wait for the jobs to complete, to save the execution time.
Can we just leave the jobs running, and don't wait for them to complete?

Reply:
@lizzha , Yes I've considered this. 
In case of managed disks, there will not be any jobs waiting.
In case of unmanaged disks, We wait till the VMs + Disks are deleted. After that, resource group cleanup starts, and LISAv2 continues...

If we don't wait, then job may get terminated, and VHD's will not be removed.
And, This block of code executes at the end of LISAv2 only.

Till now, I've observed that it may wait upto 2 minutes.